html/pfappserver/root/static/admin/config/connection_profile.js

```diff
@@ -277,8 +277,18 @@ function updateFilterMatchInput(type_input, keep) {
         match_input_template = $('#default_filter_match');
     }
     if ( match_input_template.length ) {
-        changeInputFromTemplate(match_input, match_input_template, keep);
-        if (type_value == "switch") {
+        changeInputFromTemplate(match_input, match_input_template, keep_value);
+        if(type_value == "ssid") {
+            $(type_input.next()).on('change', function(event){
+                /* Replace SELECT with INPUT */
+                select_element = $(event.currentTarget);
+                select_element.replaceWith('<input type="text" id="'+select_element.attr('id')+'" name="'+select_element.attr('name')+'" value="'+select_element.val()+'" data-required="'+(select_element.attr('required')||'0')+'" placeholder="'+select_element.attr('placeholder')+'">');
+                if(select_element.val() == '') {
+                    type_input.next().focus();
+                }
+            });
+        }
+        if(type_value == "switch") {
```

Comment:
`else if` would be more appropriate here.

Reply:
Deprecated in https://github.com/inverse-inc/packetfence/pull/2771/commits/b04723b22d2b5c6800edf242375d6f727d15bb07